extensions/statistics/locale/en.yml

```diff
@@ -9,12 +9,16 @@ flarum-statistics:
 
     # These translations are used in the Statistics dashboard widget.
     statistics:
-      active_users_text: "{count} active"
```

Comment:
This is dead code I guess? can't see it used anywhere

Reply:
Yeah, I couldn't either